content/en/_index.md

```diff
@@ -104,19 +121,37 @@ to interact with a returned object programmatically by printing some example
 attributes to the console.
 
 {{< tabpane langEqualsHeader=true >}}
- {{< tab header="Java" >}}
+  {{< tab header="Java" >}}
     public static void main(String[] args) {
         // TODO(developer): Replace these variables before running the sample.
         String projectId = "my-project-id";
         String filePath = "path/to/image.png";
-        inspectImageFile(projectId, filePath);
+        exampleSnippet(projectId, filePath);
     }
 
     // This is an example snippet for showing best practices.
     public static void exampleSnippet(String projectId, String filePath) {
         // Snippet content ...
     }
-{{< /tab >}}
+  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Required values are passed as arguments to the main() function.
+    // Example declarations are included (but commented out) in the snippet.
+    function main(projectId, filePath) {
+      // [START example_snippet]
```

Comment:
I don't think we want region tags here - they should encapsulate the whole file of whatever this sample is.

Reply:
I'd argue region tags should **not** show the `main()` method, but I'm curious as to what other Node folks (@bcoe?) have to say.

Reply:
The goal is that users need to be able to copy and paste the snippet directly from the website and run it from whatever environment they are using. The point of the `main` is to provide a single entrypoint and streamline their execution, even if they aren't using a fancy IDE - they can just execute from the command line after editing the file with VIM or whatever they are using.

Reply:
Point taken. I presume we should also include this part?

> if (require.main === module) {
>      // Run the snippet
>      main();
>  }

Reply:
Yes, I think so.

Reply:
Reopening this since having the region tags here negates the purpose of this structure (that users can right click to run from an IDE). I tried out [outline in this comment](https://github.com/GoogleCloudPlatform/samples-style-guide/pull/28/files#r826420331) the thinking behind the structure that's proposed. I think maybe this sample as is doesn't work with the `exports.exampleSnippet = exampleSnippet;` at the bottom (unless JS has funkier scopes than I remember).

Is the hang up here on just `main()`? [In this PR](https://github.com/GoogleCloudPlatform/samples-style-guide/pull/41) I suggested I tried to clarify that it doesn't need to be _named_ main unless the language requires it. Would it be better to call it something like `runExampleSnippet` to make clear it's a stub for running `exampleSnippet`? Would that be more idiomatic for Node? cc @sofisl for her input as well

I'd think the pattern makes it really easy to run the samples and has a lot of value, but if you feel like it's at odds with being idiomatic for Node.js let me know.